### PR TITLE
Fix default config file path for run tool.

### PR DESCRIPTION
**Pull request by jhendrixMSFT**
Remove a hard-coded assumption on the location of the run tool, using the
current working directory instead.  Removed some duplicate code.

### CODE REVIEW

**Comment by jhendrixMSFT** [comment]
CC @weshaggard @maririos

**Comment by maririos** [comment]
One of the ideas of run.exe is that we should be able to run it in individual projects too.
Using `GetCurrentDirectory` will enforce to always pass the location of the config.json file if we are not running from the root directory

**Comment by jhendrixMSFT** [comment]
Using the wrapper script this shouldn't be a problem though as we can either set the working directory or the path to the config file.  Am I missing something?

**Comment by maririos** [comment]
That is true, we could add the call of the config.json file in the wrapper scripts.

**Comment by maririos** [comment]
Could you do it when updating the BT version on the repos?

**Comment by jhendrixMSFT** [comment]
Which repos need to be updated?  Presumably we can update them before updating their version of build tools, case in point I've updated this PR with the necessary change to run.cmd.

File: src/Run/Executor.cs [hunk 1]
```diff
@@ -25,24 +24,15 @@ public Executor()
...
-            if (configFile == null)
-            {
-                string executorDirectory = Path.GetDirectoryName(typeof(Executor).GetTypeInfo().Assembly.Location);
-                configFilePath = Path.GetFullPath(Path.Combine(executorDirectory, @".."));
-            }
-            else
-            {
-                configFilePath = Path.GetDirectoryName(configFile);
-            }
+            configFilePath = Path.GetDirectoryName(configFile);
```
Comment (weshaggard) [inline src/Run/Executor.cs]: Why wouldn't we actually accept the full config file path as opposed to hardcoding that the file must be called config.json

Comment (jhendrixMSFT) [inline src/Run/Executor.cs]: Good point, I can make that change.

File: src/Run/Executor.cs [hunk 2]
```diff
@@ -25,24 +24,15 @@ public Executor()
         {
             SettingParameters = new Dictionary<string, string>();
             CommandParameters = new Dictionary<string, Dictionary<string, string>>();
-            string executorDirectory = Path.GetDirectoryName(typeof(Executor).GetTypeInfo().Assembly.Location);
-            configFilePath = Path.GetFullPath(Path.Combine(executorDirectory, @".."));
+            configFilePath = Directory.GetCurrentDirectory();
```
Comment (weshaggard) [inline src/Run/Executor.cs]: Just to make sure we don't cause a bunch of breaks I would suggest staging this, by keeping the old behavior as well as adding the new behavior. If `..\config.json` exists use it but if not look in the current directory.

Comment (jhendrixMSFT) [inline src/Run/Executor.cs]: Good idea I've made this change.

**Comment by jhendrixMSFT** [comment]
@weshaggard Any other comments?

**Comment by weshaggard** [comment]
LGTM

**Pull request merged**